lib/helpers.js

```diff
@@ -464,6 +464,8 @@ const extractMatchingPermissions = function (dumpsysOutput, groupNames, grantedS
  * @property {boolean} replace [true] - Set it to false if you don't want
  *                                      the application to be upgraded/reinstalled
  *                                      if it is already present on the device.
+ * @property {boolean} partialInstall [true] - Install apks partially. It is used for 'install-multiple'.
```

Comment:
where this option is set to true by default?

Reply:
oops